commands/clans.js

```diff
@@ -4,7 +4,7 @@ module.exports.execute = async (client, message) => {
 🔸 **The Round Table**: All things Hard Mode by **Alex**#8758
 🔸 **Bards of Academia**: All things music by **alivia_trivia**#9195
 🔸 **The Fiction Faction**: Creative Writing & Story Telling by **varrictethras**#5383
-🔸 **The Gathering**: Accountability by **nurse4truth**#0929
+🔸 **The Gathering**: Accountability by **anhero**#3771
 🔸 **The Clockwork Knights**: Productivity & Efficiency through the use of Systems by **vonKobra**#0286
```

Comment:
Since you've updated one leader, can you update vonKobra to Stoneybaby too, to fix #118 ?

```suggestion
🔸 **The Clockwork Knights**: Productivity & Efficiency through the use of Systems by **stoneybaby**#3398
```

Reply:
si!